### ISSUE TITLE
[Bug]: picker is not compatible with Safari's tab navigation

### ISSUE THREAD

**Issue by hunterloftis** [adobe/spectrum-web-components#1770]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Picker

### Expected behavior

Tab and shift+tab keypresses focus Picker elements on the page in tab order across all browsers.

### Actual behavior

Safari skips Picker elements during tab navigation (and the relevant test fails locally).

### Screenshots

![image](https://user-images.githubusercontent.com/364501/132083778-530802ef-0302-4457-bd03-7f968fda9355.png)

### What browsers are you seeing the problem in?

Safari

### How can we reproduce this issue?

1. run `yarn test`
